<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set</class>
 <widget class="QDialog" name="Set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>493</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="settingFirst">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>路径颜色</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="roadColor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>84</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>蓝色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>红色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑色</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>显示坐标信息</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>84</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>默认</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>详细</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>简略</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/resource/ok.png</normalon>
    </iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
